components/ide/jetbrains/backend-plugin/BUILD.yaml

```diff
@@ -77,9 +78,19 @@ packages:
     env:
       - JB_QUALIFIER=latest
       - NO_VERIFY_JB_PLUGIN=${noVerifyJBPlugin}
+      # TODO(hw): remove after `2024.2.*` is stable
+      - SDKMAN_DIR=/home/gitpod/.sdkman
     config:
       commands:
-        - ["./build.sh", "${__git_commit}"]
+        # TODO(hw): remove after 2024.2.* is stable
+        - ["mv", "build.gradle-latest.kts", "build.gradle.kts"]
+        - - "bash"
+          - "-c"
+          - >
+            echo java=21.0.3.fx-zulu > .sdkmanrc
```

Comment:
Is it requirement only for build time? at runtime any java version will work still with 2024.2?

Reply:
`2024.2` requires java 21

![photo_2024-07-05 13 47 39](https://github.com/gitpod-io/gitpod/assets/20944364/f302c4aa-6612-41a5-8e5b-e5417a58fb20)

Reply:
ok, just asking that we don't need to change anything in our images, java 21 then is bundled together with IJ?